docs/operator-manual/applicationset/Controlling-Resource-Modification.md

```diff
@@ -14,17 +14,49 @@ To enable dry-run, add `--dryrun true` to the ApplicationSet Deployment's contai
 
 See 'How to modify ApplicationSet container parameters' below for detailed steps on how to add this parameter to the controller.
 
-### Policy - `create-only`: Prevent ApplicationSet controller from modifying or deleting Applications
+### Managed Applications modification Policies
 
 The ApplicationSet controller supports a parameter `--policy`, which is specified on launch (within the controller Deployment container), and which restricts what types of modifications will be made to managed Argo CD `Application` resources.
 
 The `--policy` parameter takes three values: `sync`, `create-only`, and `create-update`. (`sync` is the default, which is used if the `--policy` parameter is not specified; the other policies are described below).
 
+It is also possible to set this policy per ApplicationSet.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+spec:
+  # (...)
+  syncPolicy:
+    applicationsSync: create-only # create-update, sync
+```
+
+If the controller parameter `--policy` is set, it takes precedence on the field `applicationsSync`. It is possible to allow per ApplicationSet sync policy by setting controller parameter `--allow-policy-override` (default to `false`).
+
+This does not prevent deletion of Applications if the ApplicationSet is deleted
```

Comment:
I need to read the PR more, but why can't `syncPolicy.applicationsSync=create-only` prevent Application deletion?

Reply:
It prevents the deletion in case the result of the generator removes an application. But from what I saw in the code the deletion of the applicationset deletes the application:

https://github.com/argoproj/argo-cd/blob/master/applicationset/controllers/applicationset_controller.go#L195